scripts/utils/trading_strategy_helpers.js

```diff
@@ -1002,6 +1002,7 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     const tokenInfoPromises = fetchTokenInfoFromExchange(await exchangePromise, [baseTokenId, quoteTokenId])
     const createFleetTransaction = buildDeterministicFleetOfSafes(masterAddress, fleetSize, await noncePromise)
     const bracketAddresses = await calcSafeAddresses(fleetSize, await noncePromise)
+    console.log(bracketAddresses)
```

Comment:
Was this intended? Do we not have a dedicated logger in this method?

Reply:
Whoops!